src/Appwrite/Utopia/Request/Filters/V19.php

```diff
@@ -2,23 +2,102 @@
 
 namespace Appwrite\Utopia\Request\Filters;
 
+use Appwrite\Query as AppwriteQuery;
 use Appwrite\Utopia\Request\Filter;
+use Utopia\Database\Database;
+use Utopia\Database\Query as UtopiaQuery;
+use Utopia\Database\Validator\Authorization;
 
 class V19 extends Filter
 {
     // Convert 1.6 params to 1.7
     public function parse(array $content, string $model): array
     {
-        /*
-        Uncomment with first request filter; current is just a copy of V18
-        switch ($model) {
-            case 'functions.create':
-                $content['something'] = $content['somethingElse'] ?? "";
-                unset($content['something']);
-                break;
+        $content = $this->manageSelectQueries($content, $model);
+
+        return $content;
+    }
+
+    protected function manageSelectQueries(array $content, string $model): array
+    {
+        $isDatabaseModel = \str_starts_with($model, 'databases.');
+        $isTargetOperation = \in_array($model, ['databases.listDocuments', 'databases.getDocument']);
+
+        if (! $isDatabaseModel || ! $isTargetOperation) {
+            return $content;
+        }
+
+        $hasWildcard = false;
+        if (! isset($content['queries'])) {
+            $hasWildcard = true;
+            $content['queries'] = [AppwriteQuery::select(['*'])];
```

Comment:
Why do we add * here? Don't we just remove it and re-add later?

Reply:
This is for the case there are no queries, which includes both getDocument and listDocuments. We remove later only for listDocs.